rskj-core/src/main/java/co/rsk/RskContext.java

```diff
@@ -154,7 +155,7 @@ public class RskContext implements NodeContext, NodeBootstrapper {
 
     private static final String CACHE_FILE_NAME = "rskcache";
 
-    private final CliArgs<NodeCliOptions, NodeCliFlags> cliArgs;
```

Comment:
do you think this field can stay `final`? seems it's being initialised only during construction.

Reply:
I was having a sonarqube complain on that one. But will see how to fix it.